express/__init__.py

```diff
@@ -32,7 +34,7 @@ def __init__(self, parser_name=None, *args, **kwargs):
         else:
             self.parser = None
 
-    def _get_parser_class(self, parser_name: str):
+    def _get_parser_class(self, parser_name: str) -> Union[Type[BaseProperty], None]:
```

Comment:
A preferred shorthand for `Union[T, None]` is `Optional[T]`, available in typing as `from typing import Optional`

Reply:
Didn't know about that one, updated!